src/python/tests/appengine/handlers/jobs_test.py

```diff
@@ -81,6 +81,21 @@ def test_post(self):
     resp = self.app.post_json('/', {'page': 4})
     self.assertListEqual([], [item['id'] for item in resp.json['items']])
 
+  def test_fuzzers_result(self):
```

Comment:
Also add a test for job update. You can mock out `update_mappings_for_job` if that makes things easier and just assert that it was called with the right values.

Reply:
yes, I have added a particular test for this and I will try to make another PR to cover the non-unit tested handlers to get more practice on working with mocks. pretty cool stuff!